pallets/fee/Cargo.toml

```diff
@@ -75,6 +75,7 @@ runtime-benchmarks = [
     "frame-benchmarking",
     "frame-support/runtime-benchmarks",
     "frame-system/runtime-benchmarks",
+    "currency/runtime-benchmarks",
```

Comment:
need the `"currency/runtime-benchmarks"` to be able to run tests with the feature `runtime-benchmarks`.